README.md

```diff
@@ -1,7 +1,8 @@
 <img src="Documentation/operator_logo_metering_color.svg" height="125px"></img>
 
 # Notice
-Metering Operator is no longer under active development. If you are interested in taking over development of this project 
```

Comment:
Ah it looks like openshift 4.8 release notes mention a bump in kube-state-metrics, and our e2e suite is failing as we haven't migrated to the next set of metrics: https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-monitoring-kube-state-metrics-v2.0.0-upgrade.